### DISCUSSION TITLE
Knowlg Friday's - Session 2

### DISCUSSION THREAD

**Discussion by vinukumar-vs** [Sunbird-Knowlg/Community#14] [General Discussions]
Folks,

Welcome to our second Knowlg Fridays session - a community connect event where we learn and grow together as a community!

This session will be a continuation of our first session. This session will be more of a workshop/hands-on session.
[
![image](https://user-images.githubusercontent.com/4496096/171340570-f797f5a3-301a-41fa-b93b-55c3c5926b28.png)
](https://docs.google.com/forms/d/1tN3UOmwuUSJYflX9IYAN_wfxwk4VsawFeofMPOELO18/viewform?edit_requested=true)

### Asset architecture

- How to create a new schema for an object? (Movie object explained in the previous session)
- How the schema will get published?
- How content-service will read the published schema's?
- Basic CRUP API's on newly published schema? ( Movie object)
- How the data will sync to ElasticSearch?
- How to use search API to retrieve data?

**Speakers:** @aks30 @AmiableAnil @joffinjoy

Join webinar - [Zoom link](https://us06web.zoom.us/j/82109956783?pwd=bjltelN4MHk2eHdpV1lGT29hbFlKUT09)
Meeting ID: 821 0995 6783
Passcode: 002581

@pramodkvarma, @mathewjpallan, @pkranga, @Prerna789, @rahul-kankaria, @coolbung, @aks30, @vijiurs, @ChakshuGautam, @kiranharidas187, @pritamps, @chitranshu-keshav @rahul101001000 @coolbung
@SanthoshVasabhaktula, @vrayulu, @surendrasinghs, @pallakartheekreddy, @krgauraw, @luckynvdu @reshmi-nair, @sharathkashyap, @gandham-santhosh, @kameshbhr, @AmiableAnil, @Jayaprakash8887, @anandp504, @maheshkumargangula, @rhwarrier

**Comment by joffinjoy** [comment]
@vinukumar-vs @Krishnaj20  Please share recorded session video link for this session.

**Reply by Krishnaj20** [reply]
Session recording details:
https://us06web.zoom.us/rec/share/JFWt-rNyrOQLelfvCfjIi7XVMhl2d1RxA3raxX2SxtTjSsN6bbfSyFnWKXgeYm-E.4HH9mz1wWzHyHaE6?startTime=1654248723000
Passcode: xs.&DZ2D

**Reply by baljeetsingh** [reply]
Thank you for sharing, however password doesn't seem to work.

**Reply by vinukumar-vs** [reply]
@baljeetsingh Password is right. I am able to open using the same above password

**Reply by baljeetsingh** [reply]
thanks @vinukumar-vs 
It didn't work initially, but then it worked if I am signed into zoom account

**Reply by vinukumar-vs** [reply]
Please find the below deck/presentation used for this session
[Knowlg Friday - Session 2](https://docs.google.com/presentation/d/1KEdSMr1UYVxY4upqLQS7rXI302LW_bNAIR5aVHv3jHc/edit?usp=sharing)
